documentation/modules/security/proc-installing-certs-per-listener.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-installing-certs-per-listener-{context}']
+= Installing your own CA certificates for listeners
+
+You can install your own certificates and public keys for Kafka brokers on a per-listener basis. These certificates can be signed by an external Certificate Authority (CA), such as your company's internal CA or a public CA, rather than the cluster or clients CAs (which are internal to Strimzi). Installing per-listener certificates is supported for two types of listeners:
```

Comment:
I am not sure about mentioning Strimzi here. Should we use `{ProductName}`?

Reply:
Good spot - thank you!